install.rst

```diff
@@ -229,7 +229,30 @@ To avoid having to install the database at all, you can run both it and the serv
     # volumes:
     #   - "./pgdata:/var/lib/postgresql/data"
 
-Go into the directory where you saved this file and run :code:`docker-compose up`. You will see the logs of both the database and PostgREST, and be able to access the latter on port 3000.
+Go into the directory where you saved this file and run :code:`docker-compose up`.
+You will see the logs of both the database and PostgREST, and be able to access the latter on port 3000.
+
+Go into the directory where you saved this file and run :code:`docker-compose up`.
```

Comment:
Redundant line, maybe this will be the last change and then I'll merge.

Reply:
Did you get it?